simdistserve/README.md

```diff
@@ -48,10 +48,31 @@ Ideally you should get the following result:
 Best per GPU rate: 1.56
 Best config: pp_cross=1, tp_prefill=2, pp_prefill=1, tp_decode=1, pp_decode=1
 ```
-
+### Ratio search
+search the best config of the ratio between prefill and decode instances:
```

Comment:
Given the parallel strategy of prefill and decoding instances, search for the best config ratio M:N.